.stylelintrc.json

```diff
@@ -1,97 +1,37 @@
 	"declaration-block-no-shorthand-property-overrides": null,
-	"font-family-name-quotes": null,
+	"declaration-block-single-line-max-declarations": null,
 	"font-family-no-duplicate-names": null,
 	"function-url-quotes": null,
-	"function-comma-newline-after": null,
-	"function-comma-newline-before": null,
-	"function-max-empty-lines": null,
-	"function-name-case": null,
-	"function-parentheses-newline-inside": null,
-	"function-parentheses-space-inside": null,
-	"function-whitespace-after": null,
-	"hue-degree-notation": null,
 	"indentation": null,
-	"length-zero-no-unit": null,
-	"linebreaks": null,
-	"max-empty-lines": null,
+	"keyframes-name-pattern": null,
 	"max-line-length": null,
-	"no-empty-first-line": null,
-	"no-duplicate-selectors": null,
-	"no-missing-end-of-source-newline": null,
 	"no-descending-specificity": null,
-	"property-case": null,
-	"property-no-vendor-prefix": null,
-	"rule-empty-line-before": null,
-	"scss/double-slash-comment-whitespace-inside": null,
+	"no-duplicate-selectors": null,
+	"scss/at-extend-no-missing-placeholder": null,
+	"scss/at-import-partial-extension": null,
 	"scss/at-mixin-pattern": null,
+	"scss/comment-no-empty": null,
 	"scss/dollar-variable-pattern": null,
+	"scss/double-slash-comment-whitespace-inside": null,
 	"scss/no-global-function-names": null,
-	"scss/dollar-variable-empty-line-before": null,
-	"scss/dollar-variable-colon-space-before": null,
-	"scss/dollar-variable-colon-space-after": null,
-	"scss/dollar-variable-colon-space-after": null,
-	"scss/at-extend-no-missing-placeholder": null,
-	"scss/comment-no-empty": null,
 	"scss/operator-no-unspaced": null,
-	"scss/at-import-partial-extension": null,
-	"scss/double-slash-comment-empty-line-before": null,
 	"selector-class-pattern": null,
-	"selector-attribute-quotes": null,
-	"selector-attribute-brackets-space-inside": null,
-	"selector-attribute-operator-space-after": null,
-	"selector-attribute-operator-space-before": null,
-	"selector-combinator-space-after": null,
-	"selector-combinator-space-before": null,
-	"selector-descendant-combinator-no-non-space": null,
-	"selector-id-pattern": null,
-	"selector-list-comma-newline-after": null,
-	"selector-list-comma-newline-before": null,
-	"selector-list-comma-space-after": null,
-	"selector-list-comma-space-before": null,
-	"selector-max-empty-lines": null,
-	"selector-no-vendor-prefix": null,
-	"selector-not-notation": null,
-	"selector-pseudo-class-case": null,
-	"selector-pseudo-class-parentheses-space-inside": null,
-	"selector-pseudo-element-case": null,
-	"selector-pseudo-element-colon-notation": null,
-	"selector-type-case": null,
-	"shorthand-property-no-redundant-values": null,
-	"keyframes-name-pattern": null,
-	"value-keyword-case": null,
-	"value-list-comma-newline-after": null,
-	"value-list-comma-space-after": null,
-	"value-list-comma-space-before": null,
-	"value-list-max-empty-lines": null,
-	"value-no-vendor-prefix": null
+	"selector-id-pattern": null
```

Comment:
The rest of the rules are things that need to have complete manual intervention and need to be evaluated on a case by case basis. For now, we're turning them off.